### PR TITLE
Roadmap page first pass

### PR DESCRIPTION
**Pull request by yahoopete**
Need to discuss the consistent way for components to refer to roadmaps and banner nav etc, but let's get something out there for our users and we can worry about what's optimal later.

### CODE REVIEW

File: tests/app/main/views/test_index.py [hunk 1]
```diff
@@ -17,7 +17,7 @@ def test_logged_in_user_redirects_to_choose_service(
 
 
 @pytest.mark.parametrize('view', [
-    'cookies', 'trial_mode', 'pricing', 'terms', 'delivery_and_failure', 'integration_testing'
+    'cookies', 'trial_mode', 'pricing', 'terms', 'delivery_and_failure', 'integration_testing', 'roadmap'
```
Comment (leohemsted) [inline tests/app/main/views/test_index.py]: 👌

Comment (yahoopete) [inline tests/app/main/views/test_index.py]: white-kid-dance-club

File: app/templates/views/roadmap.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+{% endblock %}
+
+{% block maincolumn_content %}
+
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-large">Roadmap for GOV.UK Notify</h1>
+
+    <p>
+        Here’s a list of the new functionality we’re planning to offer through GOV.UK Notify over the next 6–9 months.
```
Comment (quis) [inline app/templates/views/roadmap.html]: 6 to 9 months

> Use ‘500 to 900’ and not ‘500-900’ (except in tables).

– https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#numbers

File: app/templates/views/roadmap.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      <ul class="list list-bullet">
+        <li>Searching templates</li>
+        <li>Sending letters (invite only)</li>
+        <li>Sending text messages to international numbers</li>
+      </ul>
+
+    <h2 class="heading-medium">May 2017</h2>
+      <ul class="list list-bullet">
+        <li>Supporting replies to text messages (invite only)</li>
+        <li>Retiring the V1 API</li>
```
Comment (quis) [inline app/templates/views/roadmap.html]: version 1 of the API

File: app/templates/views/roadmap.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+        <li>Sending individual messages via Notify interface</li>
+        <li>Restricting notifications to sociable hours</li>
+        <li>Searching for notifications by phone number or email address</li>
+      </ul>
+
+    <h2 class="heading-medium">June - August 2017</h2>
```
Comment (quis) [inline app/templates/views/roadmap.html]: June to August

File: app/templates/views/roadmap.html [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+      <ul class="list list-bullet">
+        <li>Sending letters (available for everyone)</li>
+        <li>Distributing delivery of notifications over a period of time</li>
+        <li>Allowing multiple sender names and reply–to addresses</li>
```
Comment (quis) [inline app/templates/views/roadmap.html]: hyphen not en dash

File: app/templates/views/roadmap.html [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+        <li>Sending of pre–compiled text messages and emails</li>
```
Comment (quis) [inline app/templates/views/roadmap.html]: hyphen not en dash

File: app/templates/views/roadmap.html [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+        <li>Checking mobile numbers are valid before sending</li>
+        <li>Providing non–mobile phone 2–factor authentication</li>
```
Comment (quis) [inline app/templates/views/roadmap.html]: hyphen not en dash ×2

File: app/templates/views/roadmap.html [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+        <li>Sending letters through the API</li>
+        <li>Customising the expiry time for text messages</li>
+      </ul>
+
+    <h2 class="heading-medium">September 2017 and beyond</h2>
+      <ul class="list list-bullet">
+        <li>Sending of pre–compiled letters</li>
```
Comment (quis) [inline app/templates/views/roadmap.html]: hyphen not en dash

File: tests/app/main/views/test_index.py (hunk 1 above)
Comment (quis) [inline tests/app/main/views/test_index.py]: giphy

**Comment by quis** [comment]
Gonna fix all these things now… hold tight.

**Pull request merged**